src/Aquifer.API/Endpoints/Bibles/Versification/Response.cs

```diff
@@ -0,0 +1,20 @@
+namespace Aquifer.API.Endpoints.Bibles.Versification;
+
+public class Response
+{
+    public IEnumerable<VerseMapping> VerseMappings { get; set; } = [];
+}
+
+public class VerseReference
+{
+    public required int VerseId { get; set; }
+    public required string Book { get; set; }
+    public required int Chapter { get; set; }
+    public required int Verse { get; set; }
+}
+
+public class VerseMapping
+{
+    public required VerseReference TargetBibleVerse { get; set; }
+    public required VerseReference SourceBibleVerse { get; set; }
```

Comment:
None of the other response properties use "Bible" in the names so I suggest removing them here, too, for consistency.
```suggestion
    public required VerseReference TargetVerse { get; set; }
    public required VerseReference SourceVerse { get; set; }
```